frappe/public/scss/desk/frappe_datatable.scss

```diff
@@ -87,6 +87,11 @@
 		}
 	}
 
+	.dt-cell__resize-handle {
+		right: -3px !important;
+		left: unset !important;
+	}
+
```

Comment:
This is required because Frappe's build system generates RTL styles automatically, but datatable already has RTL styles which means that RTL styles rules defined in datatable are no-op 🤦🏻‍♂️. Redefining here makes it work.